staging/cse/windows/azurecnifunc.ps1

```diff
@@ -408,6 +408,14 @@ function New-ExternalHnsNetwork
         Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_MANAGEMENT_IP_NOT_EXIST -ErrorMessage "Failed to find $managementIP after creating $externalNetwork network"
     }
     Write-Log "It took $($StopWatch.Elapsed.Seconds) seconds to create the $externalNetwork network."
+
+    Write-Log "Log network adapter info after creating $externalNetwork network"
+    Get-NetIPConfiguration -AllCompartments -ErrorAction Ignore
+
+    $dnsServers=Get-DnsClientServerAddress -ErrorAction Ignore
+    if ($dnsServers) {

```

Comment:
If $dnsServers is null, should we return any error message here?

Reply:
good question. I do not log any error for below reasons:
1. it will always return a dns server
2. we will not see below log for dns server but see the result of above netipconfiguration and it also indicates that $dnsServers is null in case it occurs